grouper/fe/templates/group.html

```diff
@@ -40,7 +40,7 @@
 
     <!-- Show the join button if the user isn't already a member and the group
          isn't invite-only. -->
-    {% if not current_user.my_role(members) and group.canjoin != "nobody" %}
+    {% if not current_user['is_approver'] and group.canjoin != "nobody" %}
```

Comment:
huh. you want `current_user_role` vs `current_user`. i.e. `current_user_role['is_approver']`. i guess selenium would have caught this...